examples/issues/node_casbin_261_model.conf

```diff
@@ -0,0 +1,15 @@
+[request_definition]
```

Comment:
`node_casbin_261_model.conf` I don't like the 261 naming. Can you name it in a normal way as other model files? Or do Golang have any such model files to use?

Reply:
do you have the idea of this name?

Reply:
See Golang: https://github.com/casbin/casbin/search?q=batchEnforce&type=issues

Reply:
What do you think of using model text and string adapter?

Reply:
I think if Golang has an similar example model & policy, we should also use the same. Does Go have it?

Reply:
@nodece you can name it from the meaning, use a similar naming way as other files. `node_casbin_261_model.conf` is just a bad name, as it's coming from Node-Casbin GitHub issue #261, this is not good.

Reply:
`go-casbin` does not have a similar model and policy, `node_casbin_261_model.conf` looks clear, please let me know if you have more good names.

Reply:
@nodece 

rbac_g2_model.conf

rbac_g2_policy.conf